src/Augurk.UI/Components/SequenceDisplay.razor

```diff
@@ -0,0 +1,45 @@
+<!--
+ Copyright 2021, Augurk
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+@using System.Web
+@inject IJSRuntime jsRuntime
+
+<div @ref=Div class="sequence">
+
+</div>
+@if(!String.IsNullOrWhiteSpace(Label))
+{
+<span class="label">@Label</span>
```

Comment:
Indentation?

Reply:
Might have been a bit hasty once it finally worked :P